src/app/[locale]/viewer/_components/SubmodelsOverviewCard.tsx

```diff
@@ -17,11 +17,12 @@ import { useNotificationSpawner } from 'lib/hooks/UseNotificationSpawner';
 import { showError } from 'lib/util/ErrorHandlerUtil';
 import { performSearchSubmodelFromAllRepos } from 'lib/services/MultipleRepositorySearch/MultipleRepositorySearchActions';
 
-export type SubmodelsOverviewCardProps = { readonly smReferences?: Reference[]; readonly isLoading?: boolean };
+export type SubmodelsOverviewCardProps = { readonly smReferences: Reference[] };
```

Comment:
The Overview Card is loading its data itself, so we dont need the isLoading param from outside